hack/dockerized

```diff
@@ -149,9 +149,11 @@ if [ -n "$($KUBEVIRT_CRI ps --format '{{.Names}}' | grep ${BUILDER}-bazel-server
     fi
 fi
 
+opts="--cap-add CAP_SYS_ADMIN -v /sys/fs/cgroup/:/sys/fs/cgroup/"
```

Comment:
How dangerous is this in terms of security?

Reply:
Ops it's a leftover, removed